contribs/gmf/less/mobile.less

```diff
@@ -122,12 +125,24 @@ span.twitter-typeahead {
       content: '\f00d';
       color: grey;
       font-family: FontAwesome;
-      font-size: 1.5em;
+      font-size: 2em;
```

Comment:
Cross is really big. I would suggest

`font-size: 1.8em;`

Please note: Adjusting icon sizes without seeing all icons resized is pretty hard